### ISSUE TITLE
Update Gcov and Clang versions to latest

### ISSUE THREAD

**Issue by msaju** [gluster/project-infrastructure#90]
Gcov and Clang versions are on old versions.
Need to update to latest.

**Comment by mscherer** [comment]
Can you explain a bit more about gcov ? the lcov test is run on centos 7, but until the tests are working on Centos 8 and/or Fedora 32, we can't move it to a newer version. For clang, I have submitted a patch to move that to F32.

**Comment by mscherer** [comment]
Reopening, the issue was closed by automation github side (seems that GH is not smart enough to understand english when i say "partial fix" :p ).

**Comment by msaju** [comment]
Ok, will raise a separate ticket for gcov as soon as the centos 8 tests passes.

**Comment by msaju** [comment]
Noticed that the latest CLANG scan is giving "Zero" bugs. Is it WIP ?

**Comment by mscherer** [comment]
No, seems to be a bug. I will take a look, I guess something changed in a subtle way that make jenkins fail without error.

**Comment by mscherer** [comment]
09:49:35 configure: error: libuuid is required to build glusterfs

so it seems that if mock fail to build, our test continue normally. I guess we need a way to make it fail in a more visible way. We could fail if there is 0 bugs found, but that seems wrong (like, what if one day, we have perfect code ?)

**Comment by mscherer** [comment]
So https://build.gluster.org/job/clang-scan/1197/ work, "87 bugs found" so 4 more than last one. I guess that mean it is picking new errors, so that's good.

**Comment by msaju** [comment]
> Ok, will raise a separate ticket for gcov as soon as the centos 8 tests passes.

The Cent OS 8 regression is passing now. Is it required to create a new ticket for the GCOV?.
